### ISSUE TITLE
[issue]: How to delete or hide the version information and URL in the lower left corner

### ISSUE THREAD

**Issue by cookieiz** [ventoy/Ventoy#2448]
### Official FAQ

- [X] I have checked the official FAQ.

### Ventoy Version

master

### What about latest release

Yes. I have tried the latest release, but the bug still exist.

### Try alternative boot mode

Yes. I have tried them, but the bug still exist.

### BIOS Mode

Legacy BIOS Mode

### Partition Style

MBR

### Disk Capacity

64GB

### Image file checksum (if applicable)

None

### What happened?

![image](https://github.com/ventoy/Ventoy/assets/111113097/74add580-3cf7-48c4-9665-e793212179e7)

**Comment by wiz64** [comment]
why do you want to hide it ?
version info can be useful

**Comment by ccapy** [comment]
yes but it can also  be ugly if you're trying to make a sleek theme

**Comment by tefod-zz** [comment]
> why do you want to hide it ? version info can be useful
I believe, that reason for removing a "branding" should doesn't matter so much as long as you comply with the GPL3.
Its the grub `vtoy_license` environemt variable, which is been set hardcoded somewhere in the sources.

**Comment by tefod-zz** [comment]
> yes but it can also be ugly if you're trying to make a sleek theme

Exactly. Just updated my own custom skin and it does not really fit into it. Does anybody find a workaround to change/disable the branding?

**Comment by steve6375** [comment]
There is an easy way to suppress the version string. Details are in my eBook 'Getting started with Ventoy'.

**Comment by steve6375** [comment]
Your quite right. I've removed it.

**Comment by harukodi** [comment]
"ventoy_color": "#FFFFFF00" add this to ventoy.json seems to work
